### PR TITLE
feature: Notifications when app is not in focus and not in compact mode

### PR DESCRIPTION
**Pull request by ehguo**

### CODE REVIEW

File: app/utils/notifications.util.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import { ipcRenderer } from 'electron';
+import { GET_IS_FOCUSED } from '../electron/events';
+
+export const triggerNotification = phase => {
+  let isFocused = ipcRenderer.sendSync(GET_IS_FOCUSED);
+  let title, body, notification;
+  switch (phase) {
+    case 0:
```
Comment (chengsieuly) [inline app/utils/notifications.util.js]: For each case, you can use the enums from here:
https://github.com/builtwithluv/ZenFocus/blob/master/app/containers/enums.js to be more clear what each case means

**Comment by chengsieuly** [comment]
Just one comment, when that is fixed, then LGTM

File: app/utils/notifications.util.js (hunk 1 above)
Comment (ehguo) [inline app/utils/notifications.util.js]: Not sure how to use those - something like:

import { Phases } from '../containers/enums';
...
...
switch (phase) {
  case Phases[focus]:
etc
?

Comment (chengsieuly) [inline app/utils/notifications.util.js]:
```
switch (phase) {
 case Phases.FOCUS: {
   ...
 }

  case Phases.SHORT_BREAK:

... etc
}
```

**Pull request merged**